src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -131,6 +131,14 @@ class ReportActionItemMessageEdit extends React.Component {
         Report.saveReportActionDraft(this.props.reportID, this.props.action.reportActionID, '');
         toggleReportActionComposeView(true, this.props.isSmallScreenWidth);
         ReportActionComposeFocusManager.focus();
+        // The listener below scrolls to the last comment once the keyboard is hidden to make sure the comment is not hidden and 
+        // does not go outside the viewable area in the scroll view. This only applies to the last/latest comment in the report.
+        if (this.props.index === 0) {
+            const keyboardDidHideListener = Keyboard.addListener('keyboardDidHide', () => {
```

Comment:
Keyboard is not imported, please commit that too!

Reply:
my bad, not being able to test due to the login issue is impacting this......but still it is a mistake on my end, will check carefully.